src/PaperG/FirehoundBlob/BlobInterface.php

```diff
@@ -13,6 +13,4 @@ public function toArray();
      * @param $array array
      */
     public function fromArray($array);
-
-    public function validate();
```

Comment:
What was the conclusion about the validate method?  I notice FacebookBlob lacks any validate method.